### PR TITLE
deps: bump deps.

### PR DESCRIPTION
**Pull request by bingcicle**
- bumped `sway` dependencies to `0.24.5`
- bumped some `fuels` deps according to https://github.com/FuelLabs/sway/pull/2896

### CODE REVIEW

**Review by JoshuaBatty — APPROVED** [review]

File: src/ops/deploy/op.rs [hunk 1]
```diff
@@ -158,11 +158,7 @@ async fn create_signed_contract_tx(
     let coin_witness_index = 1;
 
     let inputs = signer_wallet
-        .get_asset_inputs_for_amount(
-            AssetId::default(),
-            DEFAULT_SPENDABLE_COIN_AMOUNT,
-            coin_witness_index,
-        )
+        .get_asset_inputs_for_amount(AssetId::default(), 1_000_000, coin_witness_index)
```
Comment (eureka-cpu) [inline src/ops/deploy/op.rs:161]: Should we be getting rid of the const here?

Comment (kayagokalp) [reply src/ops/deploy/op.rs:161]: I think it is removed from the `fuels_core` so we need to remove that. See https://github.com/FuelLabs/sway/pull/2896#discussion_r983856142 for more detail.

Comment (eureka-cpu) [reply src/ops/deploy/op.rs:161]: Hm, should we maybe put it behind a local const then for readability? eg const SPENDABLE_COIN_AMOUNT = 1_000_000

Comment (eureka-cpu) [reply src/ops/deploy/op.rs:161]: After reading through other PRs I see in some context local consts are used and other times not, up to you @bingcicle !

**Pull request merged**